### ISSUE TITLE
Next release planning (1.14.0)

### ISSUE THREAD

**Issue by intgr** [typeddjango/django-stubs#1334]
Tracking a few regressions in [version 1.13.2](https://github.com/typeddjango/django-stubs/releases/tag/1.13.2), we should probably get out a release quickly once these are resolved.

* #1335
  * Fixes #1333
  * Fixes #1336
* https://github.com/typeddjango/django-stubs/pull/1331
  * Fixes #1330
* #1345
  * Fixes #1327

We can update to mypy 0.991 thanks to @flaeppe, and will call the next version 1.14.0.

* #1329
  * Unblocked #1260 
  * Fixes #1261

Also some nice to have PRs still waiting for community reviewers:
* #1309
* #1308
* #1315

**Comment by intgr** [comment]
Lots of good stuff already merged to master, also besides the ones linked above.

As it stands, the desired behavior in #1327 is still undecided. IMO we can delay this. Additional opinions welcome.

Opened PR for release #1343.

**Comment by ngnpope** [comment]
Have reviewed #1315 - just one nit.
Will try to review #1309 and #1308 later today.

Would still like to see a solution to #1276 as that is a blocker for me upgrading as it introduced a ridiculous number of errors... Will need to have a dig into that...

**Comment by intgr** [comment]
All the reported regressions from previous version have been fixed, I'll go ahead with the release.

**Comment by intgr** [comment]
Go get it while it's hot: https://pypi.org/project/django-stubs/1.14.0/

Release notes: https://github.com/typeddjango/django-stubs/releases/tag/1.14.0

**Comment by sobolevn** [comment]
Thank you for your awesome work! 🎉

**Comment by intgr** [comment]
Thanks also to you @sobolevn for keeping django-stubs maintained for so long previously.
